content/case_histories/bookpurnong/setup.rst

```diff
@@ -2,3 +2,39 @@
 
 Setup
 =====
+
+The survey area under investigation is along Murray River in the Bookpurnong Irrigation District of the Riverland region of South Australia (Figure :numref:`booky-location`). The irrigation on the river bank (high land) has recharged the groundwater and formed a hydraulic gradient towards the floodplain (Figure :numref:`booky-hydro`). The groundwater flow from the high land to the the floodplain brings in substantial amount of natural saline water, causing tree health problems (Figure :numref:`deadtree`). In order to preserve the ecosystem, Bookpurnong Salt Interception Scheme (SIS) is implemented to experiment the water management methods that use pumping and flooding, with a hope of replacing the saline water with fresh water. 
+
+
```

Comment:
Figure are not referenced, numbered. I've found them too small too (need to click and expand to see where the area actually is)